### ISSUE TITLE
How to enable batchnorm and dropout features for training?

### ISSUE THREAD

**Issue by rajmadhan** [facebookresearch/Detectron#217]
I couldn't find any example configuration file using the batchnorm and dropout parameters.
Could anyone please show me how to add these parameters in the config file for training.

**Comment by rbgirshick** [comment]
BN is not supported when finetuning models for detection due to the small minibatch size. You may be interested in the recently release Group Norm support https://github.com/facebookresearch/Detectron/tree/master/projects/GN.

For dropout, we have not found it useful, but you can manually modify the code to add it.
